instruqt/airgap-embedded/08-deploying-a-new-version/setup-jumpbox

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This set line ensures that all failures will cause the script to error and exit
+set -euxo pipefail
+
+HOME_DIR=/home/replicant
+
+# get manifests to use for updating
+git clone --depth 1 --filter=blob:none --sparse --branch instruqt-airgap-embedded https://github.com/replicatedhq/kots-field-labs
+cd kots-field-labs
+git sparse-checkout init
+git sparse-checkout set instruqt/airgap-embedded
+mv instruqt/airgap-embedded/vendor/manifests ${HOME_DIR}
+cd ..
+rm -rf kots-field-labs
```

Comment:
Why not do a `replicated release download`? Now this has the branch hard coded, so will be difficult to maintain.

Reply:
I'm not downloading the release because I don't have any credentials to do it with here.

Reply:
Yes you do 😛 
```
json=$( jq -n -c --arg email "$INSTRUQT_PARTICIPANT_ID@replicated-labs.com" --arg password "$INSTRUQT_PARTICIPANT_ID" '$ARGS.named' )
token=$(curl -H "Content-Type: application/json" --request POST -d "$json" https://id.replicated.com/v1/login | jq -r ".token")
apiToken=$( jq -n -c --arg name "instruqt" --argjson read_only false '$ARGS.named' )
access_token=$(curl -s -H "Content-Type: application/json" -H "Authorization: $token" --request POST -d "$apiToken" https://api.replicated.com/vendor/v1/user/token | jq -r ".access_token")
```

ok. I just wrote this. But as you know the participant id, and the app slug, it should be possible to set this up and use the replicated cli

Reply:
Cool!